src/cdk/v2/destinations/dynamic_yield/procWorkflow.yaml

```diff
@@ -0,0 +1,177 @@
+bindings:
+  - name: EventType
+    path: ../../../../constants
+  - path: ../../bindings/jsontemplate
+    exportAll: true
+  - path: ../../../../v0/destinations/dynamic_yield/config
+  - name: removeUndefinedAndNullValues
+    path: ../../../../v0/util
+  - path: ../../bindings/jsontemplate
+
+steps:
+  - name: validateInput
+    template: |
+      let messageType = .message.type;
+      $.assert(.message.type, "message Type is not present. Aborting message.");
+      $.assertConfig(.destination.Config.apiKey, "Invalid Api Key");
+      $.assert(messageType in {{$.EventType.([.IDENTIFY, .TRACK])}}, 
+        "message type " + messageType + " is not supported")
+
+  - name: messageType
+    template: |
+      .message.type.toLowerCase()
+
+  - name: validateInputForTrack
+    description: Additional validation for Track events
+    condition: $.outputs.messageType === {{$.EventType.TRACK}}
+    template: |
+      $.assert(.message.event, "event_name could not be mapped. Aborting.")
+
+  - name: commonFields
+    description: |
+      Builds common fields in destination payload.
+    template: |
+      let commonFields = .message.().({
+        "user": { "id": {{{{$.getGenericPaths("userId")}}}}},
+        "session": {"custom": {{{{$.getGenericPaths("sessionId")}}}}},
+        "context":{"device":{"ip": .context.ip}}
+      });
+      commonFields.session.custom = String(commonFields.session.custom);
+      $.removeUndefinedAndNullValues(commonFields)
+
+  - name: validateCommonFields
+    template: |
+      const userId = $.outputs.commonFields.user.id;
+      const sessionId = $.outputs.commonFields.session.custom;
+      const ip = $.outputs.commonFields.context.device.ip;
+      $.assert(userId, "userId is required");
+      $.assert(sessionId, "sessionId is required");
+      $.assert(ip, "ip is required");
+
+  - name: prepareIdentifyPayload
+    condition: $.outputs.messageType === {{$.EventType.IDENTIFY}}
+    steps:
+      - name: propertiesFields
+        template: |
+          const email = .message.().(
+            {{{{$.getGenericPaths("email")}}}};
+            );
+          const hashEmail = .destination.Config.hashEmail;
+          hashedEmail = hashEmail ? SHA256(email) : email;
```

Comment:
@koladilip Am I using it correctly? Can you please check once?

Reply:
https://github.com/rudderlabs/rudder-workflow-engine/pull/67 check this you will get complete idea.

summary: you need to use like `$. SHA256(email)`

Reply:
Updated